### ISSUE TITLE
Disappearing Mobs

### ISSUE THREAD

**Issue by chrisdomaub-dev** [gamefreedomgit/Maelstrom#5760]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (http://wowhead.com/)
[//]: # (cata-twinhead.twinstar.cz)

**Description:**

Northrend, Storm Peak, The Foot Steppes Bug.
Some mobs disappears when you get in range, but very visible and clickable when not in range.
1. Tracker Thulin
2. Ice Steppe Rhino

Basically I cannot proceed with my quests since I am not able to kill these certain mobs since it disappears when I get in range (melee or long range)

**How to reproduce:**
Basically I've been getting all quests that I can in Storm Peak, Northrend.

**How it should work:**

**Database links:**

**Comment by Revly** [comment]
Maybe u was another phase

**Comment by Waroxx-QA** [comment]
That's phasing, you have to complete some quests to be able to see those mobs in that specific area.
